docs/DeveloperGuide.md

```diff
@@ -749,49 +767,227 @@ _Customization of Contacts_
 </div>
+### Launch and exiting the app
+
+Download the latest `jar` file from SociaLite [release page](https://github.com/AY2122S1-CS2103T-F11-4/tp/releases), and copy it into an empty folder that you have access to read and write.
+
+1. Launching
+   1. Double-click on the `jar` file icon in your OS' GUI
+   
+   	Expected: the GUI for SociaLite app shows up with a set of sample contacts.
+   
+   2. Locate the directory using your terminal and enter `java -jar socialite.jar`.
+   
+   	Expected: ditto.
+   
+2. Exiting
+
+	While SociaLite is running:
+
+	1. Click on the `X` button on the window (top-left for macOS, top-right for Windows).
+
+		Expected: the Socialite app quits normally.
+
+	2. While SociaLite is running, type `exit` in the command box of the GUI and hit `Enter`.
+
+		Expected: ditto.
+
+### Adding a contact
+
+While SociaLite is running, try out the following test cases in sequence:
+
+1. Enter `add n/aa p/12345678` in the command box.
+
+	Expected: A new contact card with name "aa" and phone number "12345678" is created. The contact card is most likely to be appearing at the top, since SociaLite arranges the contact cards according to their names alphabetically.
+
+2. Enter `add n/ab p/12345678` in the command box.
+
+	Expected: SociaLite shows error message "A contact with the same phone number already exists in the contact list!".
+
+3. Enter `add n/aa` in the command box.
+
+	Expected: SociaLite shows error message, indicating invalid command format given. This is because name and phone fields are compulsory for the addition of contact.
+
+4. Enter `add n/aa p/87654321` in the command box.
+
+	Expected: A new contact card with name "aa" and phone number "87654321" is created. SociaLite allows repetitive names in the contact list, but uses phone number as the unique identifier for a contact.
+
+5. Enter `add n/John Appleseed p/13572468 fb/john.appleseed twitter/morethanfifteenchars`.
+
+	Expected: SociaLite shows error message, indicating that the telegram username does not follow their naming convention.
+
+### Editing a contact
+
+Assuming all test cases in the previous section were carried out, and that the first contact card in the list is "aa", and there are only 10 contacts stored in SociaLite:
+
+1. Enter `edit 1 n/abc` in the command box.
+
+	Expected: the first contact card's name changes to "abc". It might not be the first contact card in the list due to alphabetical ordering.
+
+2. Enter `edit 100 n/abc` in the command box.
+
+	Expected: SociaLite shows error message, indicating invalid person index provided.
+
+3. Enter `edit 2` in the command box.
+
+	Expected: SociaLite shows error message, indicating at least one field to edit must be provided.
+
+4. Enter `edit 1 fb/hello` in the command box.
+
+	Expected: the first contact card in SociaLite should have a Facebook icon showed up at the bottom, with its handle being "hello".
```

Comment:
Can change to 'Expected: the first contact card in SociaLite should now display a Facebook icon at the bottom, with the handle "hello".'